tests/webview/visual/test_homepage_collections.py

```diff
@@ -0,0 +1,422 @@
+from time import sleep
+from applitools import MatchLevel
+from tests import markers
+
+from pages.webview.home import Home
+
+APP_NAME = 'CNX HOMEPAGE BOOKS3'
```

Comment:
is this the best name? What does this mean?

Reply:
I changed it to webview_homepage. In my other test case I work on, I would name it webview_pages as it will check pages